### ISSUE TITLE
Can't add AutoMapper 4.x to Xamarin.Forms project

### ISSUE THREAD

**Issue by mplogas** [AutoMapper/AutoMapper#856]
When trying to add AutoMapper 4.x to a Xamarin.Forms project using NuGet it fails with the following error message
```Fehler beim Hinzufügen des Verweises zu "System.Collections". Stellen Sie sicher, dass sich das Verweisziel im globalen Assemblycache befindet.```
Translation: *Failed to add reference to 'System.Collections'. Please make sure that it is in the Global Assembly Cache.*

Installation of AutoMapper 3.3.1 using NuGet works fine.

OS: Windows 10
IDE: VS.NET 2015
NuGet: 3.1.1.0
Xamarin.Forms: 1.4.4.6392

**Comment by jbogard** [comment]
Can you determine which assembly VS tries to reference? Is it the PCL?

**Comment by mplogas** [comment]
I'm afraid ```Update-Package AutoMapper -ProjectName IoT.Mobile.Droid -Verbose``` wasn't really verbose at all.

```
    + CategoryInfo          : NotSpecified: (:) [Update-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.UpdatePackageCommand
```

I'm using Shared Projects, they target v2.0.50727 of the .NET runtime.

**Comment by onovotny** [comment]
Hi @mplogas, are you sure you're running on .NET 2.0? I didn't think Xamarin Forms even supported that? What version of Android are you targeting?

Do you have the latest Xamarin tools installed?

Do you have a repro solution where it fails?

**Comment by mplogas** [comment]
@onovotny it seems using a Xamarin.Forms Shared project, the default BCL is .NET 2.0.5, at least system.dll is marked with that version. When using a Xamarin.Forms PCL I can freely choose any BCL target I have installed on my machine.

I'm targeting Android API level 22, have the latest Xamarin tools installed (at least there aren't any updates available) and you can reproduce the issue by creating a new Xamarin.Forms Shared project in Visual Studio 2015 and try to add AutoMapper using NuGet.

**Comment by jbogard** [comment]
Xamarin Forms Shared projects don't reference any BCL I didn't believe. You can't add NuGet package references there.

I created in VS 2015 and a Xamarin.Forms shared project, added a NuGet package reference to AutoMapper 4.0.4 and it all compiled just fine:

![image](https://cloud.githubusercontent.com/assets/104498/9595051/6a83e6b6-5029-11e5-98de-2ee4ffe4b2f4.png)

**Comment by mplogas** [comment]
I didn't try to add it to the SharedLib itself, but to the .Droid and .iOS projects. 
Anyway, I switched to PCL and the NuGet import worked fine.

**Comment by jbogard** [comment]
Weird, I tried the shared project and it all worked, *shrug*

**Comment by jbogard** [comment]
OK this one was a malformed nuspec, it's fixed in 4.1.1 (now on the prerelease feed).
